.goreleaser.yaml

```diff
@@ -189,14 +189,14 @@ nfpms:
 
 snapcrafts:
   - name: opentofu
-    publish: true
+    publish: "{{ .Env.STABLE_RELEASE }}"
```

Comment:
I have a feeling we need to keep this as a boolean, not move it to a string. But I could be wrong. I'm not sure how goreleaser handles this

Reply:
In my opinion, it should work as a boolean variable, as `STABLE_RELEASE` is being assigned the negation of inputs.prerelease in `release.yml`. However, we do need to fully test this first.

Reply:
But you're wrapping it in quotes, that is changing the type here from boolean to string.

Reply:
We discussed this in a call, we think that we can drop the `publish` property completely here.

Reply:
removed the publish changes